components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -114,12 +113,21 @@ class BrowserToolbar @JvmOverloads constructor(
         }
 
     /**
-     *  Set/Get the site security icons (usually a lock or globe icon). It uses a pair of drawables
-     *  which represent the insecure and secure colours respectively.
+     * Set/Get the site security icon (usually a lock and globe icon). It uses a
+     * [android.graphics.drawable.StateListDrawable] where the "enabled" state represents the secure
```

Comment:
I like the  use of `StateListDrawable`, but I'm not quite sure about using already defined states. We used something similar on the [tracking protection icons by using custom states](https://github.com/mozilla-mobile/android-components/blob/2cc3aed61c5daf5a4184e1a3fffd311bd99371df/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt#L226).  I think the custom states is a good approach in the long run as states grows (Not sure if it's the right one now). There are some proposals for adding new  states to the security indicator related to the media feature I don't know if it's final cc:@pocmo . 

<img width="1089" alt="Screen Shot 2019-08-07 at 3 12 12 PM" src="https://user-images.githubusercontent.com/773158/62875986-009e6980-bcf2-11e9-9bea-5f4dda82a50c.png">

More custom states examples

https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/TrackingProtectionIconView.kt

https://github.com/mozilla-mobile/android-components/blob/master/components/browser/toolbar/src/main/res/drawable/mozac_tracking_protection_state_list.xml

Reply:
[Media feature mockups](https://app.goabstract.com/share/8bc72f1f-46ef-4cb0-a0cc-58f43993e90d)